package.json

```diff
@@ -41,7 +41,8 @@
     "compile:current": "cross-env MODE=production npm run build && electron-builder build --config .electron-builder.config.cjs",
     "test": "npm run test:unit && npm run test:e2e:smoke",
     "test:unit": "npm run test:main && npm run test:preload && npm run test:preload-docker-extension && npm run test:renderer && npm run test:tools && npm run test:extensions",
-    "test:e2e:smoke": "cross-env NODE_ENV=development MODE=development DEBUG=pw:browser npm run build && xvfb-maybe vitest run tests",
+    "test:e2e:smoke": "cross-env NODE_ENV=development MODE=development DEBUG=pw:browser npm run build && xvfb-maybe vitest run tests/src/e2e-smoke.spec.ts",
+    "test:e2e:full": "cross-env NODE_ENV=development MODE=development DEBUG=pw:browser npm run build && xvfb-maybe vitest run tests/src/e2e-full.spec.ts",
```

Comment:
To me, pulling test looks a smoke test, not a 'full' one

Reply:
if you want to rename e2e file to smoke tests, it's a separate PR

Also it's not clear if smoke means a dedicated file or a dedicated pattern.

Like: is it e2e-smoke.spec.ts top level file that would call other test suites or is it a pattern like all files named `*.smoke.spec.ts`

Then I would assume full will launch all files from the tests folder while smoke tests would run only the files named `*.smoke.spec.ts`